files/scripts/syncd.sh

```diff
@@ -28,6 +28,19 @@ function startplatform() {
         debug "Firmware update procedure ended"
     fi
 
+    if [[ x"$sonic_asic_platform" == x"broadcom" ]]; then
+        if [[ x"$WARM_BOOT" != x"true" ]]; then
+            is_bcm0=$(ls /sys/class/net | grep bcm0)
```

Comment:
what is is_bcm0, this is unclear to me.

Reply:
is_bcm0 indicates the presence of the base netdevice created by the BRCM SDK kernel module.  If the bcm0 netdevice is present, the BRCM SDK kernel modules are loaded.  We only take action to remove and reload the kmods if they are already loaded.